src/stim/stabilizers/tableau.pybind.cc

```diff
@@ -208,6 +208,34 @@ void stim_pybind::pybind_tableau(pybind11::module &m) {
         )DOC")
             .data());
 
+    c.def(
+        "to_pauli_string",
+        [](const Tableau &self) {
+            return PyPauliString(self.to_pauli_string());
+        },
+        clean_doc_string(u8R"DOC(
+            If the tableau fixes each pauli upto sign, creates the pauli
+            string which represents the tableau.
+
+            Returns:
+                The created pauli string
+
+            Raises:
+                ValueError: The Tableau isn't equivalent to a Pauli product.
+
+            Example:
+                >>> import stim
+                >>> p = stim.PauliString("+YX_Z")
```

Comment:
It's a bit of a cop-out for the example to do the round trip thing, instead of starting from an explicit tableau. It's another good test, though.

Reply:
:+1: